paddle/fluid/operators/conv_op.cc

```diff
@@ -303,6 +320,9 @@ void Conv3DOpMaker::Make() {
       "Defaults to \"NHWC\". Specify the data format of the output data, "
       "the input will be transformed automatically. ")
       .SetDefault("AnyLayout");
+  AddAttr<bool>("force_fp32_output",
+                "(bool, default false) Only used in mkldnn INT8 kernel")
+      .SetDefault(false);
```

Comment:
Calibration tool will set the value. By default it is false, which means INT8 convolution output is INT8. However, it is allowed to set output FP32.

Reply:
> Calibration tool will set the value

If it only set in runtime, it could not be added here. You can use following codes to insert the attr at runtime.
https://github.com/PaddlePaddle/Paddle/blob/55af1168614edd1d9a0b9077304141f4e458c92b/paddle/fluid/framework/ir/is_test_pass.cc#L45-L46

Reply:
Currently , our calibration tool is implemented on python, which will save such flags to model offline, we will do this runtime after we enable C PASS calibration.